Apps/WebClient/src/ClientApp/src/components/navmenu/HeaderComponent.vue

```diff
@@ -84,15 +87,16 @@ export default class HeaderComponent extends Vue {
 
     private lastScrollTop = 0;
     private static minimunScrollChange = 2;
+    public notificationButtonClicked = false;
 
-    private get userName(): string {
+    public get userName(): string {
```

Comment:
should these actually be public or private? I saw the warnings before, but I think private is fine too.

Reply:
@MikeLyttle has changed a few. I had discussed with @MikeLyttle about this issue and it did not seem like an issue to change.  @MikeLyttle comment please.

Reply:
I'm looking into an experimental setting for Volar that might get rid of the problems without us having to change the accessibility.

Reply:
I have reverted these changes as I see there is a PR for Volar fix.